rkt/rkt.go

```diff
@@ -181,7 +183,9 @@ func init() {
 
 func getTabOutWithWriter(writer io.Writer) *tabwriter.Writer {
 	aTabOut := new(tabwriter.Writer)
+
```

Comment:
reference? I can't find anything (looking for "logic", "group",
"statements", "newlines") in https://golang.org/doc/effective_go.html - and
indeed all of the code examples there seem to suggest otherwise

On Tue, Apr 19, 2016 at 1:53 PM, Tamer TAS <notifications@github.com> wrote:

> In rkt/rkt.go
> <https://github.com/coreos/rkt/pull/2453#discussion_r60216890>:
>
> > @@ -181,7 +183,9 @@ func init() {
> >
> >  func getTabOutWithWriter(writer io.Writer) *tabwriter.Writer {
> >  	aTabOut := new(tabwriter.Writer)
> > +
>
> I'm adding a new lines to logically group statements suggested by Effective
> Go docs.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/2453/files/bbb8bf451a4cabc67e74f03103df18b24a2a5219#r60216890>
>